### ISSUE TITLE
body class for all non ie browsers

### ISSUE THREAD

**Issue by frederfred** [brightcove-archive/zencoder_html5-boilerplate-for-wordpress#2]
The current version adds the class "ie6" to the body to all browsers that aren't internet explorer. The body element should look like this instead.

 <body <?php body_class(); ?>>

**Comment by aoberoi** [comment]
I think this was meant to change line 58 of header.php to:

```
<!--[if (gt IE 9)|!(IE)]><!--> <body <?php body_class(); ?>> <!--<![endif]-->
```

**Comment by ucc-xx** [comment]
yes, that is an issue which should be resolved and yes that is most likely what frederfred meant.
